app/hooks/useRedirectToContacts.tsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from "react";
+import { useRouter } from "next/router";
+import { getContactsPageRoute } from "pageRoutes";
+/**
+ *  Hook that returns true if the project revision being viewed is not the latest project revision,
+ *  the latest project revision id is not null, and shouldRedirect is true.
+ *  False otherwise.
```

Comment:
```suggestion
 *  Hook that redirects to the contacts list page, if the contact being viewed is committed and the `preventRedirect` flag is false.
 *  If the redirect is happening, then the hook returns true, and false otherwise.
```